src/library-authoring/common/data/constants.js

```diff
@@ -54,6 +54,14 @@ export const ROUTES = {
   },
 };
 
+export const BLOCK_TYPE = {
```

Comment:
can we use `PAGE` here? "block_type" in this context developers would assume to mean type of xblock, and thus this name is confusing.